.github/workflows/pr_assignee.yml

```diff
@@ -1,19 +1,19 @@
 name: Assign PR to creator
 
-# Due to GitHub token limitation, only able to assign org members not authors from forks.
-# https://github.com/thomaseizinger/assign-pr-creator-action/issues/3
-
 on:
   pull_request:
     types: [opened]
     branches-ignore:
       - l10n_dev
 
+permissions:
+  pull-requests: write
+
 jobs:
   automation:
     runs-on: ubuntu-latest
     steps:
     - name: Assign PR to creator
-      uses: thomaseizinger/assign-pr-creator-action@v1.0.0
+      uses: toshimaru/auto-author-assign@v2.1.1
       with:
```

Comment:
do we still need the token?

Reply:
It will use the GITHUB_TOKEN by default if not specified, so we can remove it.

Reply:
I can't make the suggestion commit because that line hasn't changed and I'm on my phone, so let's ignore for now, doesn't make much of difference.